TopsortAnalytics/src/main/java/com/topsort/analytics/banners/BannerView.kt

```diff
@@ -0,0 +1,36 @@
+package com.topsort.analytics.banners
+
+import android.content.Context
+import android.widget.ImageView
+import coil.load
+import com.topsort.analytics.Analytics
+import com.topsort.analytics.model.Placement
+import com.topsort.analytics.model.auctions.EntityType
+
+class BannerView(
+    context: Context,
+    config: BannerConfig,
+    path: String,
+    location: String,
+    clickCallback: (String, EntityType) -> Unit
```

Comment:
Is this the callback needed for navigation after clicking?

Reply:
yes, I have to change the signature according to the article you sent me, though

Reply:
I'd change it to `onClick`, or `onTap` in that case